### ISSUE TITLE
Add ccache to default image ?

### ISSUE THREAD

**Issue by GuillaumeSeren** [nicolasbock/ebuildtester#217]
Hello,
I know ebuildtester use the default stage3 image, so maybe it is an issue for gentoo BTS,
but if the ccache support could already installed in the container,
because installing ccache take a little while (and will not be cached :/ ).

**Comment by stkw0** [comment]
Yeah, ebuildtester uses official gentoo images which afaik none of which contains dev-util/ccache. Given that ebuildtester supports the use of binhosts a quick workaround would be to set-up a small binhost which contains pre-compiled packages that you may want inside the container, dev-util/ccache for instance.
